changelog/4340.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Data products from `~sunpy.net.dataretriever.NOAAIndicesClient` and
+`~sunpy.net.dataretriever.NOAAPredictClient` have been updated to download
+new JSON files. The old text files which the data used to come in no longer
+exist.
+
+Both `sunpy.timseries.sources.NOAAIndicesTimeSeries` and
+`sunpy.timseries.sources.NOAAPredictIndicesTimeSeries` have been updated to
+support the new JSON files. Loading the old text files is still supported,
+but support for this will be removed in a future version of sunpy.
```

Comment:
Add something about the differences in the columns?

Reply:
I can add it.

Reply:
Updated.